dashboard/src/App.test.js

```diff
@@ -12,8 +12,6 @@ const AppWrapper = () => {
   );
 };
 
-test('renders learn react link', () => {
+test("renders learn react link", () => {
   render(<AppWrapper />);
-  const linkElement = screen.getByText('Explore');
-  expect(linkElement).toBeInTheDocument();
 });
```

Comment:
It's on hold for now!

Reply:
This presents a dilemma:  as a rule, we should be resistant to having "placeholders" in the code, particularly ones which cost us execution time like this.  They are clutter, and they tend to "rot" and cause other hassles.

My recommendation is that, unless you're intending to add something to this test for this PR, just go ahead and remove the whole function.  It will be easy enough to add back in when you are ready for it to actually do something.  If you really want to keep it in the code, then please add a comment explaining the situation, so that future developers will know why the no-op code is there and so that they will have some context when deciding when and whether they can delete it.

If you're simply concerned about it being forgotten, then create an issue for it in Jira and remove it from the code.

Reply:
Removed the placeholder test case. Created [PBENCH-1038](https://issues.redhat.com/browse/PBENCH-1038) to track